source/slang/slang-ast-expr.h

```diff
@@ -139,6 +139,12 @@ class StringLiteralExpr : public LiteralExpr
     FIDDLE() String value;
 };
 
+FIDDLE()
+class ArrayWithGenericSizeExpr : public Expr
```

Comment:
This naming is confusing. The Expr node represents an option that is to create an array from a single element value. It doesn't matter if the array type has generic size or not. This should just named `MakeArrayFromElementExpr`.

Reply:
Done